Formula/bali-phy.rb

```diff
@@ -21,7 +23,7 @@ class BaliPhy < Formula
   def install
     flags = %w[-C build install]
     # Reduce memory usage for Circle CI
-    flags += %w[-j 4] if ENV["CIRCLECI"]
+    flags << "-j2" if ENV["CIRCLECI"]
```

Comment:
`CIRCLECI` should now be `HOMEBREW_CIRCLECI`. Sorry for the back and forth.
Can you please try `-j4`? Before the PR upstream was merged, it was defaulting to `-j8`, which used too much memory. `-j4` may work.

Reply:
```suggestion
    flags << "-j4" if ENV["HOMEBREW_CIRCLECI"]
```

Reply:
Done.